File: Data Structures & Algorithms/maximum-ascending-subarray-sum/submission-1.java
class Solution {
    public int maxAscendingSum(int[] nums) {
        int maxsum = Integer.MIN_VALUE;
        int sum = 0;
        int left = 0;
        int right = 0;
        while(left<=right && left<nums.length && right<nums.length)
        {
            sum+=nums[right];
            right++;
            if(right<nums.length && nums[right]<=nums[right-1])
            {
                maxsum = Math.max(maxsum, sum);
                sum = 0;
                left = right;
            }
            maxsum = Math.max(maxsum,sum);
        }
        return maxsum;
    }
}

File: Data Structures & Algorithms/is-palindrome/submission-1.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string str = "";
        for(char c:s)
        {
            if(isalnum(c))
            {
                str+=tolower(c);
            }
        }
        int left=0, right = str.length()-1;
        while(left<right)
        {
            if(str[left]==str[right])
            {
                left++;
                right--;
            }else
            {
                return false;
            }
        }
        return true;
    }
};
